android/app/src/fdroid/java/org/fossasia/openevent/core/location/OSMapFragment.java

```diff
@@ -20,10 +20,10 @@
 import org.fossasia.openevent.R;
 import org.fossasia.openevent.common.ConstantStrings;
 import org.fossasia.openevent.common.utils.SharedPreferencesUtil;
-import org.fossasia.openevent.data.Event;
-import org.fossasia.openevent.data.Microlocation;
-import org.fossasia.openevent.data.Session;
-import org.fossasia.openevent.data.repository.RealmDataRepository;
+import org.fossasia.data.Event;
+import org.fossasia.data.Microlocation;
+import org.fossasia.data.Session;
+import org.fossasia.openevent.repository.RealmDataRepository;
```

Comment:
Package doesn't need to change. Package should remain `org.fossasia.openevent`

Reply:
Okay I will make those changes